app/ts/constants.ts

```diff
@@ -1,3 +1,35 @@
-export const NETWORKS: { [chainId: string]: { simulationRelay: string, submissionRelay: string, blockExplorer: string, rpcUrl: string }} = {
-	'1': { simulationRelay: 'https://flashbots-cors-proxy.dark-florist.workers.dev/', submissionRelay: 'https://rpc.titanbuilder.xyz', blockExplorer: 'https://etherscan.io/', rpcUrl: 'https://rpc.dark.florist/flipcardtrustone' },
+type Network = { networkName: string, simulationRelay: string, submissionRelay: string, blockExplorer: string, rpcUrl: string, chainId: string, blockExplorerApi: string }
+
+export const MAINNET = {
+	networkName: 'Mainnet',
+	simulationRelay: 'https://flashbots-cors-proxy.dark-florist.workers.dev/',
+	submissionRelay: 'https://rpc.titanbuilder.xyz',
+	blockExplorer: 'https://etherscan.io/',
+	rpcUrl: 'https://rpc.dark.florist/flipcardtrustone',
+	chainId: `0x${1n.toString(16)}`,
+	blockExplorerApi: 'https://api.etherscan.io'
+}
```

Comment:
Why not define this inline and then reference it with `NETWORKS[1n]` when you need it?

Alternatively, we could do something like:
```ts
export const MAINNET_ID = 1n
export const SEPOLIA_ID = 11155111n
```
Then you could access it with `NETWORKS[MAINNET_ID]`.

Could also do an enum if you wanted, though I'm not a huge fan in TS.

Reply:
I don't need other networks like that, just the default mainnet one. I guess make an object out of it, and then convert that object to a map, but that sounds more complex